### ISSUE TITLE
[Asset] Thumbnails with specific image area not working

### ISSUE THREAD

**Issue by daves-weblab** [pimcore/pimcore#5806]
### Expected behavior
When using an document image tag which renders a set thumbnail everything works fine unless a specific image area is selected via the context menu.

### Actual behavior
The image tag produces a temporary thumbnail (image-thumb__{assetId}__{thumbnail-name}\_auto\_{hash}) configuration, due to the selected image area, which does not work.

Thumbnail with name '{thumbnail-name}_auto_abdbcd55933359f02146c92b934ea10a' doesn't exist in the PublicServicesController.

### Steps to reproduce

1. Create an image tag in an areabrick or the document itself. 
2. Provide a thumbnail configuration that already exists. 
3. Select an image area of the image tag (Select specific area of image).
4. Visit the page itself, the image won't load anymore.

### Code example
```php
{{ pimcore_image("my-image", {
    thumbnail: "my-thumbnail",
    lowQualityPlaceholder: true,
    hidetext: true
}) }}
```

**Comment by dvesh3** [comment]
@daves-weblab cannot reproduce on https://demo.pimcore.fun/ use "Image" brick on any content page and try to reproduce it.

Image brick is using existing thumbnail:
![image](https://user-images.githubusercontent.com/5137917/74538451-8ff43600-4f3c-11ea-89fb-8cb94e8cc1a4.png)

**Comment by dvesh3** [comment]
@daves-weblab any chance to look at it?

**Comment by daves-weblab** [comment]
hi @dvesh3, I just updated to the latest 6.5.1 tag, which fixed the issue, sry for the inconvenience

**Comment by dvesh3** [comment]
@daves-weblab 👍  can you please update the affected version in description? Thanks!

**Comment by brusch** [comment]
should be fixed by #5886

**Comment by brusch** [comment]
It only happens under certain circumstances: auto format, WebP variant and @2x together with custom cropping.
